hat/assets/js/apps/Iaso/domains/app/translations/fr.json

```diff
@@ -899,6 +900,8 @@
     "iaso.permissions.tooltip.polio_budget": "Consultation du processus d’approbation du budget and actions possibles en fonction du rôle assigné à l’utilisateur par équipe",
     "iaso.permissions.tooltip.polio_budget_admin": "Consultation du processus d’approbation du budget and actions possibles en fonction du rôle assigné à l’utilisateur par équipe. Pouvoirs Admin: si besoin, imposer une étape du budget indépendamment des conditions pré-établies",
     "iaso.permissions.tooltip.polio_config": "Gestion de l’entrée de données sur les campagnes Polio, des groupes de campagnes, du calendrier, des résultats LQAS/IM. Pouvoirs Admin: Création de campagnes Test, informations de debug LQAS/IM, configuration par pays (emails automatiques, langue par pays, gestion des équipes de budget)",
+    "iaso.permissions.tooltip.polio_vaccine_authorizations_admin": "Manage polio vaccine authorizations - Read and Write",
+    "iaso.permissions.tooltip.polio_vaccine_authorizations_read_only": "Manage polio vaccine authorizations - Read Only",
```

Comment:
That's strange, translations in this file should be in French, no?

Reply:
Yeah that's correct. This happened when I ran **python3 scripts/update_trads.py** to fix translation error and I forgot to correct it.